src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/DWMColorization.cs

```diff
@@ -0,0 +1,682 @@
+    {
+        if (brightnessFactor > 100f || brightnessFactor < -100f)
+        {
+            throw new ArgumentOutOfRangeException(nameof(brightnessFactor));
+        }
+
+        if (saturationFactor > 100f || saturationFactor < -100f)
+        {
+            throw new ArgumentOutOfRangeException(nameof(saturationFactor));
+        }
+
+        if (luminanceFactor > 100f || luminanceFactor < -100f)
+        {
+            throw new ArgumentOutOfRangeException(nameof(luminanceFactor));
+        }
+
+        (float hue, float rawSaturation, float rawBrightness) = ToHsv(color);
+
+        (int red, int green, int blue) = FromHsvToRgb(
+            hue,
+            ToPercentage(rawSaturation + saturationFactor),
+            ToPercentage(rawBrightness + brightnessFactor)
+        );
+
+        if (luminanceFactor == 0)
+        {
+            return Color.FromArgb(color.A, ToColorByte(red), ToColorByte(green), ToColorByte(blue));
+        }
+
+        (hue, float saturation, float rawLuminance) = ToHsl(Color.FromArgb(color.A, ToColorByte(red), ToColorByte(green), ToColorByte(blue)));
+
+        (red, green, blue) = FromHslToRgb(hue, saturation, ToPercentage(rawLuminance + luminanceFactor));
+
+        return Color.FromArgb(color.A, ToColorByte(red), ToColorByte(green), ToColorByte(blue));
+    }
+
+    /// <summary>
+    /// Allows to change the brightness by a factor based on the HSV color space.
+    /// </summary>
+    /// <param name="color">Input color.</param>
+    /// <param name="factor">The value of the brightness change factor from <see langword="100"/> to <see langword="-100"/>.</param>
+    /// <returns>Updated <see cref="System.Windows.Media.Color"/>.</returns>
+    private static Color UpdateColorBrightness(Color color, float factor)
+    {
+        if (factor > 100f || factor < -100f)
+        {
+            throw new ArgumentOutOfRangeException(nameof(factor));
+        }
+
+        (float hue, float saturation, float rawBrightness) = ToHsv(color);
+
+        (int red, int green, int blue) = FromHsvToRgb(hue, saturation, ToPercentage(rawBrightness + factor));
+
+        return Color.FromArgb(color.A, ToColorByte(red), ToColorByte(green), ToColorByte(blue));
+    }
+
+    /// <summary>
+    /// Gets <see cref="System.Windows.Media.Color"/> brightness based on HSV space.
+    /// </summary>
+    /// <param name="color">Input color.</param>
+    public static double GetBrightness(Color color)
+    {
+        (float _, float _, float brightness) = ToHsv(color);
+
+        return (double)brightness;
+    }
+
+    /// <summary>
+    /// Converts the color values stored as HSV (HSB) to RGB.
+    /// </summary>
+    private static (int R, int G, int B) FromHsvToRgb(float hue, float saturation, float brightness)
+    {
+        var red = 0;
+        var green = 0;
+        var blue = 0;
+
+        if (AlmostEquals(saturation, 0, 0.01f))
+        {
+            red = green = blue = (int)(((brightness / 100f) * _byteMax) + 0.5f);
```

Comment:
Can all the code here, as well as the code under this method, that uses `+0.5` for rounding be replaced with `Math.Round(value, MidpointRounding.AwayFromZero)`?

Reply:
As suggested, we have removed all the calculation related code, defaulted the variations to `systemAccent` as a fallback.